### ISSUE TITLE
Only check external links daily

### ISSUE THREAD

**Issue by ashawley** [scala/docs.scala-lang#1125]
Seth suggested moving the fix in https://github.com/scala/scala-lang/pull/942 that modified Travis to check external links only on a scheduled basis, but still check internal links with all git activity.

This repo doesn't use Travis.  [Drone doesn't appear to intrinsically support cron jobs](http://readme.drone.io/questions/how-to-schedule-builds/).  There would need to be a similar facility to port it here.

**Comment by SethTisue** [comment]
@jvican is this an issue you've faced/solved in any of the Scala Center's other Drone builds?

**Comment by jvican** [comment]
No, never. Adding a cron job in our machine shouldn't be an issue, but I'm not sure this is the way to go... now we're bothered by it and we at least fix them. With such a tool, I'm not sure if they would even get fixed...

**Comment by SethTisue** [comment]
> No, never. Adding a cron job in our machine shouldn't be an issue, but I'm not sure this is the way to go... now we're bothered by it and we at least fix them. With such a tool, I'm not sure if they would even get fixed...

this has to be weighed against the cost of routinely having spurious errors in PRs that are unrelated to the content of the PRs. that leads to CI being ignored by contributors and maintainers. this is the situation we are already in.

**Comment by SethTisue** [comment]
is there a strong reason we're using Drone instead of Travis-CI in this repo at all? generally we prefer to 1) keep the scala-lang and docs.scala-lang repos aligned technically, 2) use tools that are already familiar to open-source contributors (and under scala/* we've been using Travis-CI almost everywhere for a long time)

**Comment by rkoeninger** [comment]
FYI, an exception was added to the CI build for links in the `www.oracle.com` domain, which were failing regularly. (Issue #1168, fixed in PR #1169) It doesn't appear verifying external links is otherwise an issue at the moment.

**Comment by SethTisue** [comment]
Agree there's nothing really pressing here. (I still think it would be nice to align with the scala/scala-lang repo, though.)
